config/default/meta.yml.j2

```diff
@@ -10,6 +10,21 @@ on:
       - main
   workflow_dispatch:
 
+{% if env %}
+env:
+%(env)s
+{% else %}
```

Comment:
That's an interesting approach, if the setting is already used, do not add the inline documentation about it... then though, when adding/removing that setting the diff will be bigger as the documentation will be added/removed as well 🤔

Reply:
But we would benefit of having a smaller file :-)